lib/ls.js

```diff
@@ -389,6 +389,8 @@ const ls = async (args) => {
   const only = npm.config.get('only')
   const prod = npm.config.get('prod')
   const production = npm.config.get('production')
+  const depthCli = // retrieves only user-defined values
+    npm.config.get('depth', 'cli') || npm.config.get('depth', 'user')
```

Comment:
the user might also define it in an env var, or in a global config, or even in the builtin config. why limit it?

Reply:
sure, tbh I was just trying to find a quick way around changing the default config value but on a second thought it's much better to tweak that value instead and define new defaults for `update`, `fund`, etc

that sounds a lot like a different set of changes than the proposed fix from this PR so I should keep it in a sep commit at least 🤔

Reply:
thanks for the review @ljharb - intent is much more clear now 😊